### ISSUE TITLE
Feauture requst: Possible to download description of videos?

### ISSUE THREAD

**Issue by qnorsten** [spaam/svtplay-dl#435]
Would it be possible to add a command to also save the description of each video to a text-file?

**Comment by spaam** [comment]
is there a standard on how the file should be named and what kind of info it should contain? 
the thumbnail thing is from xbmc / kodi.

**Comment by qnorsten** [comment]
For svtplay for example I am thinking to just save everything in the description-tag to a textfile named the same as the downloaded videofile.

**Comment by spaam** [comment]
like super.cool.file.txt ?

**Comment by qnorsten** [comment]
Exactly. And then just parse the video description if it can be done without to much work.

**Comment by iwconfig** [comment]
isn't the description available via the api? I was thinking about this when dealing with ffmpeg. It can pretty easily be merged as metadata into the mp4 file. that's another issue i guess but anyways

**Comment by spaam** [comment]
@iwconfig nope, cant find it in http://www.svtplay.se/video/9894193/top-gear/top-gear-avsnitt-1?output=json or http://www.svt.se/videoplayer-api/video/1374818-001A

**Comment by spaam** [comment]
men det är i själva react koden på sidan.  under: context -> dispatcher -> stores -> VideoTitlePageStore -> data -> video -> description

**Comment by spaam** [comment]
http://www.jsoneditoronline.org/?id=11c8249357aa53555069bcbf04218319  line 119 on the right site :)

**Comment by iwconfig** [comment]
Ah! Just ja, så var det. :)
